tools/bouncer_worker/src/queues/modelQueueHandler.js

```diff
@@ -54,7 +56,30 @@ Handler.onMessageReceived = async (cmd, rid, callback) => {
 	};
 
 	try {
-		returnMessage.value = await runBouncerCommand(logDir, cmdParams);
+		const memoryReporting = { enabled: config.elastic.memoryReporting, maxMemory: 0, processTime: 0 };
+		returnMessage.value = await runBouncerCommand(logDir, cmdParams, memoryReporting);
+		if (memoryReporting.maxMemory > 0) {
+			const importFile = cmdParams[3];
+			// eslint-disable-next-line
+			const fileStats = fs.statSync(require(importFile).file);
+			const elasticBody = {
```

Comment:
as both unity and model queue need to generate this, i think it's best to have it in utils as a function.

So you just pass in the parameters, and it returns you the object.

This way, if we ever rename any of these, or if the body needs to be structured any differently, there's only 1 place to change.